File: Model/Artikal.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Delos.Model
{
    public class artikal
    {
        [Key]
        public string sifra { get; set; }
        public string dobavljac_sifra { get; set; }
        public string dobavljac { get; set; }
        public string naziv { get; set; }
        public decimal kolicina { get; set; }
        public decimal cijena_sa_rabatom { get; set; }
        public string dostupnost { get; set; }
        public DateTime? zadnje_ucitavanje { get; set; }

        public List<string> slike { get; set; }
    }
}

File: ClientApp/src/app/model/artikal.ts
export class Artikal {
  sifra: string;
  naziv: string;
  dobavljac: string;
  dobavljac_sifra: string;
  kolicina?: number;
  dostupnost: string;
  cijena_sa_rabatom?: number;
  cijena_prodajna?: number;
  cijena_mp?: number;
  zadnje_ucitavanje: Date;
  slike?: string[];
  vrste?: string[];
  kategorija: string;
}
